ci/testnet-automation.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+echo --- downloading snap from build artifacts
+buildkite-agent artifact download "solana_*.snap" .
+
+# shellcheck disable=SC1091
+source ci/upload_ci_artifact.sh
+
+[[ -n $ITERATION_WAIT ]] || ITERATION_WAIT=300
+[[ -n $NUMBER_OF_NODES ]] || NUMBER_OF_NODES="10 25 50 100"
+
+launchTestnet() {
+  echo --- setup "$1" node test
+  net/gce.sh create -n "$1" -c 2 -G "n1-standard-16 --accelerator count=2,type=nvidia-tesla-v100" -p testnet-automation -z us-west1-b
+
+  echo --- configure database
+  net/init-metrics.sh -e
+
+  echo --- start "$1" node test
+  net/net.sh start -o noValidatorSanity -S solana_*.snap
+
+  echo --- wait "$ITERATION_WAIT" seconds to complete test
+  sleep "$ITERATION_WAIT"
+
+  declare q_mean_tps="SELECT round(mean(\"sum_count\")) from (SELECT sum(\"count\") AS \"sum_count\" FROM \"testnet-automation\".\"autogen\".\"counter-banking_stage-process_transactions\" WHERE time > now() - 300s GROUP BY time(1s))"
```

Comment:
We can use single quotes and slashes to avoid all the escaping and super-long lines, eg:
```bash
declare q_mean_tps='\
  SELECT round(mean("sum_count")) FROM ( \
    SELECT sum("count") AS "sum_count" \
      FROM "testnet-automation"."autogen"."counter-banking_stage-process_transactions" \
      WHERE time > now() - 300s GROUP BY time(1s) \
  ) \
'